### PR TITLE
BurstCube Note

### LINKED ISSUE #2275: Add BurstCube JSON notices UI to Quickstart
# Description

Steps to release BurstCube notices.

# Acceptance criteria

- the quickstart form should have checkboxes for BurstCube notices.
- link to internal mission pages (draft PR for the mission pages)
- put behind feature flag until we're ready to announce

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
# Description
USWSD note for BurstCube

`Resolves #2275`

### CODE REVIEW

File: app/routes/missions.burstcube/route.mdx [hunk 1]
```diff
@@ -5,6 +5,17 @@ handle:
 
 import logo from './logo.png'
 
+import {
+  ProcessList,
+  ProcessListHeading,
+  ProcessListItem,
+  SummaryBox,
+} from '@trussworks/react-uswds'
```
Comment (lpsinger) [inline app/routes/missions.burstcube/route.mdx:13]: Remove unused imports.

```suggestion
import { SummaryBox } from '@trussworks/react-uswds'
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/missions.burstcube/route.mdx [hunk 2]
```diff
@@ -5,6 +5,12 @@ handle:
 
 import logo from './logo.png'
 
+import { SummaryBox } from '@trussworks/react-uswds'
+
+<SummaryBox heading="Warning">
+  **Information:** BurstCube Notice types are not yet in production.
+</SummaryBox>
```
Comment (lpsinger) [inline app/routes/missions.burstcube/route.mdx:12]: I think that an Alert would work better here.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/missions.burstcube/route.mdx [hunk 3]
```diff
@@ -5,6 +5,12 @@ handle:
 
 import logo from './logo.png'
 
+import { Alert } from '@trussworks/react-uswds'
+
+<Alert type="info" heading="Information" headingLevel="h4">
```
Comment (lpsinger) [inline app/routes/missions.burstcube/route.mdx:10]: "Information" is an unusual choice for heading. Consider the following instead:

```suggestion
<Alert type="warning" heading="Not yet active" headingLevel="h4">
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Comment by Vidushi-GitHub** [comment]
Screenshot 2024-05-23 at 9 26 04 AM

**Review by lpsinger — APPROVED** [review]

**Pull request merged**